### DISCUSSION TITLE
ERROR: Could not install packages due to an Environment

### DISCUSSION THREAD

**Discussion by ihuanh** [cnr-isti-vclab/TagLab#50] [Q&A] [answered]
Try installing for couple of time but still cannot install TagLab.

// ERROR: Could not install packages due to an EnvironmentError: [Errno 2] No such file or directory: 'C:\\Users\\user\\AppData\\Local\\Packages\\PythonSoftwareFoundation.Python.3.7_qbz5n2kfra8p0\\LocalCache\\local-packages\\Python37\\site-packages\\caffe2\\python\\serialized_test\\data\\operator_test\\collect_and_distribute_fpn_rpn_proposals_op_test.test_collect_and_dist.zip'

// C:\Program Files (x86)\TagLab-2022.04.21>python.exe taglab.py
Traceback (most recent call last):
  File "taglab.py", line 34, in <module>
    from PyQt5.QtCore import Qt, QSize, QMargins, QDir, QPoint, QPointF, QRectF, QTimer, pyqtSlot, pyqtSignal, QSettings, QFileInfo, QModelIndex
ImportError: DLL load failed:

and it keep mentioning 

// WARNING: You are using pip version 20.1.1; however, version 22.2.1 is available.
You should consider upgrading via the 'C:\Users\user\AppData\Local\Microsoft\WindowsApps\PythonSoftwareFoundation.Python.3.7_qbz5n2kfra8p0\python.exe -m pip install --upgrade pip' command.

Do I need to update the pip version to contiou the installation?

/nvcc --version
nvcc: NVIDIA (R) Cuda compiler driver
Copyright (c) 2005-2021 NVIDIA Corporation
Built on Sun_Mar_21_19:24:09_Pacific_Daylight_Time_2021
Cuda compilation tools, release 11.3, V11.3.58
Build cuda_11.3.r11.3/compiler.29745058_0

/python3 --version
Python 3.7.9

**Comment by Miguel-Figuerola** [comment]
Hello,

First, try to update the pip. This solved some issues for me.

I had a similar issue. I tried uninstalling python and during reinstall, I used the manual installation instead of the recommended one. I selected the "environmental variables" option. Also select "Add to Path" in the installation wizard. This worked for me, but make sure to erase all traces of python before doing a reinstall.

I have not been able to make the "train my network" function work and its been frustrating since I don't work with pytorch.

Good luck

**Comment by ihuanh** [comment, marked as answer]
It found out after I use "Powershell" to do all the commands everything are fine now!
Although it was mention in Step 0: Dependencies "You can check if they are properly installed by running the following commands in a shell (bash on Linux, **_poweshell_** on Windows)", I just miss this point at the very beginning.
